config/settings.yml

```diff
@@ -12,6 +12,35 @@ plugins:
       - Yandex
   translator_azure_subscription_key:
     default: ''
+  translator_azure_region:
+    default: 'global'
+    client: true
+    type: enum
+    choices:
```

Comment:
Aside from `global`, the valid choices are retrieved from https://docs.microsoft.com/en-us/azure/cognitive-services/translator/reference/v3-0-reference#authenticating-with-a-multi-service-resource